rebar.config

```diff
@@ -1,5 +1,6 @@
 {erl_opts, [
-     {platform_define, "^[0-9]+", namespaced_types}
+     {platform_define, "^[0-9]+", namespaced_types},
+     {platform_define, "^R[01][0-9]", no_map_type}
```

Comment:
Change to `{platform_define, "^(?!(R[01][0-9]))", supports_maps}` to avoid double negation in code.